extensions/firefox/content/pdfjschildbootstrap-enabled.js

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2014 Mozilla Foundation
```

Comment:
How is this file used at the moment? I don't see an include of it anywhere. Do you perhaps need to include it in build process like https://github.com/mozilla/pdf.js/blob/cd5acf501e9899f386bb7efc9feded7b2c6e763b/gulpfile.js#L822?